oauth_dropins/indieauth.py

```diff
@@ -0,0 +1,128 @@
+"""IndieAuth drop-in.
+
+https://indieauth.com/developers
+"""
+
+import json
+import logging
+import urllib
+import urlparse
+
+import appengine_config
+import handlers
+import models
+from webutil import util
+
+import bs4
+import requests
+from google.appengine.ext import ndb
+
+
+INDIEAUTH_URL = 'https://indieauth.com/auth'
+
+
+def discover_authorization_endpoint(me):
+  """Fetch a URL and look for authorization_endpoint Link header or
+  rel-value.
+
+  Args:
+    me: string, URL to fetch
+
+  Return:
+    string, the discovered indieauth URL or the default indieauth.com URL
+  """
+  logging.debug('trying to fetch user url "%s"', me)
+  resp = util.requests_get(me)
```

Comment:
thanks for using `util.requests_*`! btw you can drop the logging message above; the `util` methods log automatically.